src/playlist-maker.js

```diff
@@ -78,6 +78,38 @@ var playlistMaker = function(player, plist) {
     return currentIndex;
   };
 
+  // item can be either
+  //  * a string
+  //  * an array of sources, which are either strings or {src, type} objects
+  //  * a playlist item
+  playlist.contains = function contains(item) {
```

Comment:
Depends on who you're making these functions for, I guess. As it is now, this isn't very useful to the UI. An indexOf (ignoring the strictness question), would be.